index.js

```diff
@@ -138,6 +139,7 @@ function bootstrap(options) {
 
   app.use(userMiddleware);
   app.use(hofMiddleware.cookies());
+  app.use(markdown());
```

Comment:
We're almost 100% certainly going to want the markdown middleware to be loaded *before* the cookies and terms and conditions pages are routed in order to be able to use markdown for the content in those pages. I'd move the mounting above those routes.